scan_test.go

```diff
@@ -79,3 +79,20 @@ func TestParseZoneInclude(t *testing.T) {
 		}
 	}
 }
+
+func TestParseZoneTA(t *testing.T) {
+	r := strings.NewReader(` Ta 0 0 0`)
```

Comment:
This should also break with NewRR() which leads to a shorter testcase

Reply:
```Go
func TestParseZoneTA(t *testing.T) {
	rr, err := dns.NewRR(` Ta 0 0 0`)
	if err != nil {
		t.Fatalf("expected no error, but got %s", err)
	}
	if rr == nil {
		t.Fatalf(`expected a normal RR, but got nil`)
	}
}
```
what about this then?